hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -131,6 +131,7 @@ public void run() {
                             int i = 0;
                             for (NearCacheRecord record : records) {
                                 cache.remove(record.getKey());
+                                stats.incrementEvictions();
                                 if (++i > evictSize) {
```

Comment:
We might have an off-by-one error here, since we remove one entry more than the calculated `evictSize`. So when there are 100 entries in the Near Cache the `evictSize` will be `20`, but `21` entries will be removed from the Near Cache.